fabulaws/library/wsgiautoscale/templates/nginx.conf

```diff
@@ -47,6 +47,11 @@ server {
         proxy_read_timeout 30;
         proxy_connect_timeout 30;
 
+        # rewrite hosts with trailing dot to non-dot version
+        if ($http_host ~ "\.$" ){
+            rewrite ^(.*) $scheme://$host$1 permanent;
```

Comment:
I stand corrected; Karen's reminded me that while she committed the original change, the pointer to this Stack Overflow post came from me: http://stackoverflow.com/questions/15452328/nginx-redirect-domain-trailing-dot

I still think the redirect is fine -- that keeps everything nicely in sync on the client and the server.